recipes/ddsketch/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ddsketch" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ddsketch-{{ version }}.tar.gz
+  sha256: 00c7f9d376c2abe8f9290d10cb3c712cd9effcdcd4477147553813ce54c97fee
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools-scm
+  run:
+    - protobuf >=3.0.0
+    - python
+    - six
+    - typing  # [py<35]
```

Comment:
We only build py>=3.6, so this recipe can probably be noarch by removing this dependency.

Reply:
Thank you for the review ! I addressed the comments